package.json

```diff
@@ -33,32 +39,58 @@
       "height": 1
     },
     "applet": {
-      "defaults": {}
+      "defaults": {
+        "requestInterval": 60,
+        "upColor": "#00ff00",
+        "downColor": "#ff0000",
+        "desiredStatusCode": 200
+      }

```

Comment:
This is not used. To set default values use `value` attribute  of each question.  Example: 

```
      {
        "key": "requestInterval",
        "label": "Server ping frequency (seconds)",
        "help": "How often (in seconds) to ping the server URL",
        "required": false,
        "order": 20,
        "controlType": "textbox",
        "placeholder": "60"
        "value": "60"  <--------------------------
      },
```

Reply:
I just went ahead and pulled the whole config section as I wrote the applet's configuration to be self-sustaining with the same defaults as in that block.  I just for some reason thought I needed that section.  At this point the applet could run with no other configuration than a url, and that configuration item didn't change for this revision